actions/filter-files/action.yml

```diff
@@ -35,7 +40,8 @@ runs:
             throw new Error(`filter-files was called with an empty string as the "changed-files" parameter.`)
           }
           const inputFiles = JSON.parse(inputFilesRaw);
-          const invert = `${{ inputs.invert }}` == 'true';
+          const invert = ${{ inputs.invert == 'true' }};
```

Comment:
Instead of logging `invert = 'true' == 'true'` it now logs `invert = true`
Instead of `invert = 'true' == 'false'` now logs `invert = false`